packages/react-cosmos-component-playground/package.json

```diff
@@ -21,8 +24,10 @@
     "react": "^15.5.4",
     "react-cosmos-loader": "^2.0.0-beta.24",
     "react-cosmos-state-proxy": "^2.0.0-beta.24",
+    "react-cosmos-webpack": "^2.0.0-beta.25",
     "react-dom": "^15.5.4",
-    "react-querystring-router": "^2.0.0-beta.23"
+    "react-querystring-router": "^2.0.0-beta.23",
+    "react-test-renderer": "^15.6.1"
```

Comment:
No need for this, test dependencies are installed globally in the root package.json to not have to add them in every package.